app/views/threads/thread.html

```diff
@@ -478,17 +482,47 @@ <h5 ng-if="comments.length > 0" class="center">Comments</h5>
+                                                        <!--<loading-circle-small></loading-circle-small>-->
+                                                        <!--</div>-->
+                                                    </div>
 												</li>
 											</ul>
 											<!--End comment replies-3-->
-
-
+                                            <div class="action-links reply-action-links">
+                                                <a ng-if="comment.replies && comment.replies.hasMorePages" href="" ng-click="getCommentReplies(comment)">Load more replies</a>
+                                                <p style="display:inline-block" ng-if="comment.repliesBusy">Fetching replies...</p>
+                                                <!--Loading circles don't play well with ul .collection-item .avatar li =( -->
+                                                <!--<div ng-if="comment.repliesBusy" style="display: inline-block; vertical-align: middle">-->
+                                                <!--<loading-circle-small></loading-circle-small>-->
+                                                <!--</div>-->
+                                            </div>
 										</li>
 									</ul>
 									<!--End comment replies-2-->
+                                    <div class="action-links reply-action-links">
+                                        <a ng-if="comment.replies && comment.replies.hasMorePages" href="" ng-click="getCommentReplies(comment)">Load more replies</a>
+                                        <p style="display:inline-block" ng-if="comment.repliesBusy">Fetching replies...</p>
+                                        <!--Loading circles don't play well with ul .collection-item .avatar li =( -->
+                                        <!--<div ng-if="comment.repliesBusy" style="display: inline-block; vertical-align: middle">-->
+                                        <!--<loading-circle-small></loading-circle-small>-->
+                                        <!--</div>-->
+                                    </div>
 								</li>
 							</ul>
 							<!--End comment replies-->
+                            <div class="action-links reply-action-links">
+                                <a ng-if="comment.replies && comment.replies.hasMorePages" href="" ng-click="getCommentReplies(comment)">Load more replies</a>
+                                <p style="display:inline-block" ng-if="comment.repliesBusy">Fetching replies...</p>
+                                <!--Loading circles don't play well with ul .collection-item .avatar li =( -->
+                                <!--<div ng-if="comment.repliesBusy" style="display: inline-block; vertical-align: middle">-->
+                                <!--<loading-circle-small></loading-circle-small>-->
+                                <!--</div>-->
+                            </div>
 							<!--TODO FOR COMMENT RECURSION CHANGE UP TO HERE-->
```

Comment:
Que sería este TODO?

Reply:
El árbol de comentarios y respuestas debería ser recursivo, pero con angular es una mierda hacer eso, así que no quedó otra que hacer copy-pasta horrible del mismo código 5 veces para mostrar el árbol de replies. Este TODO está para que si encuentro alguna manera de hacerlo recursivo y que ande, de volar todo lo que esté entre `...COMMENT RECURSION...FROM HERE` y `...COMMENT RECURSION...TO HERE`, y dejar el primer nivel sólo

Reply:
👍